.circleci/config.yml

```diff
@@ -652,7 +652,9 @@ jobs:
   create-postgres-dump-from-genesis-dump:
     docker:
       - *defaultImage
-      - image: postgres:12.0-alpine
+      - image: postgres:12-alpine
+        environment:
+          POSTGRES_PASSWORD: password
```

Comment:
Took this from the stackrox repo, but we should probably not expose this nor make it obvious as maybe it's possible for someone who has access to the CI container to mess with the data. Any thoughts @gavin-stackrox ?

Reply:
Note: we don't actually use this password for our needs. It was just required to set this environment variable in the latest postgres 12 version